### DISCUSSION TITLE
remix-run Chat App built on Server sent Events

### DISCUSSION THREAD

**Discussion by MonuCarpenter** [remix-run/remix#4636] [Q&A] [answered]
I want to make a chat app which  will be based on the SSE but it should not use the extra node server.

**Comment by machour** [comment, marked as answer]
@monu1315 I deleted your question on the RFC for ServerEvent since it's not the appropriate place to ask. 

Tagging @rossipedia here: OP wanted you to share the code for the live chat demo you've put together if possible.

**Reply by MonuCarpenter** [reply]
Are you talking about the Chat App with nodemon , server.js file and with Eventemmiter but i dont want any extra node server bcoz i am working on an remix application i want to make a realtime chat system in my Application.
@machour thanks for your time ❤️
